bokeh/layouts.py

```diff
@@ -207,9 +213,16 @@ def layout(*args, **kwargs):
     # Make the grid
     return _create_grid(children, sizing_mode, **kwargs)
 
-def gridplot(children, sizing_mode=None, toolbar_location='above', ncols=None,
-             width=None, height=None, plot_width=None, plot_height=None,
-             toolbar_options=None, merge_tools=True):
+def gridplot(children: List[List[LayoutDOM]] | GridSpec, *,
```

Comment:
`List` seems too specific

Reply:
Possibly it is, but at this stage the type is about its shape and not the exact specific type. The exact type will need to be coordinated across the API.